d2l-localize-behavior.js

```diff
@@ -5,6 +5,8 @@ import { AppLocalizeBehavior } from './app-localize-behavior.js';
 import { formatFileSize } from '@brightspace-ui/intl/lib/fileSize.js';
 import { getDocumentLocaleSettings } from '@brightspace-ui/intl/lib/common.js';
 
+const supportedLangpacks = ['ar', 'cy', 'da', 'de', 'en', 'en-gb', 'es', 'es-es', 'fr', 'fr-fr', 'fr-on', 'hi', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-cn', 'zh-tw'];
```

Comment:
Core has this same list. I wonder if we can find some centralized place for it?

Reply:
Yeah, I thought about it for a second, but there are a number of other similar lists around. I'd like to get them all into `@d2l/intl` or something as part of the larger localization cleanup work.